### PR TITLE
0.10.3 docs update

### PR DESCRIPTION
**Pull request by warp-core**
**Feature:**

## Feature Details
Updates versions numbers from "0.10.3-alpha" to "0.10.3" and updates the date in the man page (endless-sky.6) to the planned release date of 0.10.3.
Also, updates the list of contributors in the credits file with 9 new contributors, one who I think should have been in the last release but was missed (LazerLit), and Lead, who made almost all of the current star sprites but doesn't have a GitHub account so isn't included in the auto-generated list.

### CODE REVIEW

**Comment by lumbar527** [comment]
Should this also change the changelog, or will it wait to do that?

**Comment by warp-core** [comment]
> Should this also change the changelog, or will it wait to do that?

I'm not writing the changelog for this release. @Amazinite has been working on it.

**Comment by lumbar527** [comment]
OK, thanks.

File: io.github.endless_sky.endless_sky.appdata.xml [hunk 1]
```diff
@@ -47,6 +47,32 @@
     </screenshot>
   </screenshots>
   <releases>
+    <release version="0.10.3" date="2023-9-30" type="development" urgency="low">
```
Comment (quyykk) [inline io.github.endless_sky.endless_sky.appdata.xml:50]:
```suggestion
    <release version="0.10.3" date="2023-09-30" type="development" urgency="low">
```

**Review by quyykk — APPROVED** [review]

**Pull request merged**